sdk/digitaltwins/Azure.DigitalTwins.Core/src/Azure.DigitalTwins.Core.csproj

```diff
@@ -0,0 +1,56 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <!-- Assembly props -->
+  <PropertyGroup>
+    <AssemblyTitle>Digital Twins SDK</AssemblyTitle>
+    <TargetFrameworks>$(RequiredTargetFrameworks)</TargetFrameworks>
+
+    <EnableApiCompat>false</EnableApiCompat>
```

Comment:
If there isn't one already lets please file an issue to get ApiCompat enabled after your first release.

Reply:
We copied other SDKs on this one. I'm not sure what it means. Can you elaborate what changing this to `true` does?

Reply:
See https://github.com/Azure/azure-sdk-for-net/blob/master/CONTRIBUTING.md#api-compatibility-verification